cmd/payload-testing-prow-plugin/server.go

```diff
@@ -229,6 +236,9 @@ func (s *server) handle(l *logrus.Entry, ic github.IssueCommentEvent) string {
 		}
 		messages = append(messages, message(spec, jobNames))
 	}
+
+	messages = append(messages, fmt.Sprintf("See details on %s/%s/%s\n", prPayloadTestsUIURL, builder.namespace, prpqrName))
```

Comment:
This has to be done for each `spec`.

Reply:
Why? The prpqr name will be the same. In the link the user can see all the jobs then.

Reply:
`specs` can have multiple items if the comment being processed has more than one `/payload` command.

Reply:
yes, and `prpqr` names will be different (they must be, they are kube resources), the builder is adding counter-based suffixes `-0` to the names

Reply:
Yea got it! Updated.

I left the `trigger 0 jobs ` message untouched for now